docs/reference/analysis/analyzers/stop-analyzer.asciidoc

```diff
@@ -1,273 +1,276 @@
-      "type": "word",
-      "position": 1
-    },
-    {
-      "token": "brown",
-      "start_offset": 12,
-      "end_offset": 17,
-      "type": "word",
-      "position": 2
-    },
-    {
-      "token": "foxes",
-      "start_offset": 18,
-      "end_offset": 23,
-      "type": "word",
-      "position": 3
-    },
-    {
-      "token": "jumped",
-      "start_offset": 24,
-      "end_offset": 30,
-      "type": "word",
-      "position": 4
-    },
-    {
-      "token": "lazy",
-      "start_offset": 40,
-      "end_offset": 44,
-      "type": "word",
-      "position": 7
-    },
-    {
-      "token": "dog",
-      "start_offset": 45,
-      "end_offset": 48,
-      "type": "word",
-      "position": 8
-    },
-    {
-      "token": "s",
-      "start_offset": 49,
-      "end_offset": 50,
-      "type": "word",
-      "position": 9
-    },
-    {
-      "token": "bone",
-      "start_offset": 51,
-      "end_offset": 55,
-      "type": "word",
-      "position": 10
-    }
-  ]
-}
-----------------------------
-
-/////////////////////
-
-
-The above example produces the following terms:
-
-[source,text]
----------------------------
-[ quick, brown, foxes, jumped, lazy, dog, s, bone ]
----------------------------
-
-[float]
-=== Definition
-
-It consists of:
-
-Tokenizer::
-* <<analysis-lowercase-tokenizer,Lower Case Tokenizer>>
-
-Token filters::
-* <<analysis-stop-tokenfilter,Stop Token Filter>>
-
-If you need to customize the `stop` analyzer beyond the configuration
-parameters then you need to recreate it as a `custom` analyzer and modify
-it, usually by adding token filters. This would recreate the built-in
-`stop` analyzer and you can use it as a starting point for further
-customization:
-
-[source,console]
-----------------------------------------------------
-PUT /stop_example
-{
-  "settings": {
-    "analysis": {
-      "filter": {
-        "english_stop": {
-          "type":       "stop",
-          "stopwords":  "_english_" <1>
-        }
-      },
-      "analyzer": {
-        "rebuilt_stop": {
-          "tokenizer": "lowercase",
-          "filter": [
-            "english_stop"          <2>
-          ]
-        }
-      }
-    }
-  }
-}
-----------------------------------------------------
-// TEST[s/\n$/\nstartyaml\n  - compare_analyzers: {index: stop_example, first: stop, second: rebuilt_stop}\nendyaml\n/]
-
-<1> The default stopwords can be overridden with the `stopwords`
-    or `stopwords_path` parameters.
-<2> You'd add any token filters after `english_stop`.
+[[analysis-stop-analyzer]]
+=== Stop Analyzer
+
+The `stop` analyzer is the same as the <<analysis-simple-analyzer,`simple` analyzer>>
+but adds support for removing stop words.  It defaults to using the
+`_english_` stop words.
+
+See the link:https://github.com/elastic/elasticsearch/blob/master/docs/reference/analysis/analyzers/stop-analyzer-language-support.asciidoc[Stop Analyzer Language Support] for more information about supported stop word languages

```

Comment:
This will link to the "Supported stop words by language" section below.
```suggestion
See the <<supported-stop-words>> for supported languages with links
to default stop words.
```